src/page-modules/contact/components/form/form.module.css

```diff
@@ -320,18 +321,25 @@
   color: var(--static-background-background_0-text);
   background-color: var(--static-background-background_0-background);
   border-radius: var(--border-radius-regular);
-  position: relative;
 }
 
 .modal_content > * + * {
-  margin-top: var(--spacings-large);
+  margin-top: var(--spacings-small);
 }
-
-.modal_header {
+* .modal_header {
```

Comment:
Is the wildcard necessary here; could it be `.modal_content .modal_header`?

Reply:
To be honest, I'm not sure why the wildcard is there in the first place. I remove it, as I see no difference whether it is there or not.